NEWS

```diff
@@ -27,6 +27,13 @@ Improvements
 * A failing ``expectThat`` now fails tests run with
   ``AsynchronousDeferredRunTest``.  (Jonathan Lange, #1532452)
 
+* New ``testtools.twistedsupport`` module that collects all of our Twisted
+  support code in one place, including that currently available under
+  ``testtools.deferredruntest``.  (Jonathan Lange)
+
+* New matchers for testing ``Deferred`` code: ``failed``, ``successful``, and
+  ``NO_RESULT``.  (Jonathan Lange, Tristan Seligmann, #1369134)
```

Comment:
From the outside, it's seems a little bit weird to have `failed` and `successful` but then `NO_RESULT`.  I don't know what the model is that makes that the right choice.

Also, less surprising but slightly annoying (and certainly a nit if there is such a thing), I would say that a Deferred "was successful" but I wouldn't say it "was failed".  The lack of parallelism irks slightly.  Using `failed` and `succeeded` would address this - both "the Deferred failed" and "the Deferred succeeded".